SecureSubmit.php

```diff
@@ -2013,44 +2023,26 @@ function submit_payment() {
                 }
             }
 
-            $config = new HpsServicesConfig();
-
-            $config->secretApiKey = esc_attr($skey);
-            $config->versionNumber = '1648';
-            $config->developerId = '002914';
-
-            $chargeService = new HpsCreditService($config);
-
-            $address = new HpsAddress();
+            $config = $this->setConfig($skey);
+            $address = new Address();
             $address->address = $billing_address;
```

Comment:
`HpsAddress::$address` should map to `Address:$streetAddress1`

Reply:
updated